core/src/main/java/org/bitcoinj/uri/BitcoinURI.java

```diff
@@ -224,15 +224,11 @@ private void parseParameters(Network network, String[] nameValuePairTokens) thro
                 // Decode the amount (contains an optional decimal component to 8dp).
                 try {
                     Coin amount = Coin.parseCoin(valueToken);
-                    if (network.exceedsMaxMoney(amount))
```

Comment:
removal of this line allows us to remove the `Network` parameter from `parseParameters()`

Reply:
Yes, we could.